README.markdown

```diff
@@ -101,6 +101,8 @@ To set a default Node version to be used in any new shell, use the alias 'defaul
 
     nvm alias default stable
 
+This creates a symlink to the stable version for the Node or to the verison you can mention instead of stable, so you don't have to `nvm use` everytime you start a shell session.
```

Comment:
Note, please don't open a new pull request - please reuse and rebase this one instead.